python/python/glide/config.py

```diff
@@ -293,6 +302,7 @@ def __init__(
         protocol: ProtocolVersion = ProtocolVersion.RESP3,
         pubsub_subscriptions: Optional[PubSubSubscriptions] = None,
         inflight_requests_limit: Optional[int] = None,
+        client_az: Optional[str] = None,
```

Comment:
is that necessary for the pubsub? @ikolomi can you take a look?
if so, maybe we need to add a test?